test/regressionTest.m

```diff
@@ -0,0 +1,57 @@
+function regressionTest(commitBefore,commitAfter)
+    % Produce graphical output for two commits
+
+    % Work only with a clean working tree
+    [status, cmdout] = system('git status --porcelain');
+    if ~isempty(cmdout)
+        error('regressionTest:treeNotClean','Working tree is not clean.')
+    end
```

Comment:
I would actually not suggest anything, since I dunno what the developer is up to. He might be working on a separate feature and just did not commit all changes yet.